ggml/src/ggml-sycl/common.hpp

```diff
@@ -471,6 +472,19 @@ static __dpct_inline__ float warp_reduce_max(float x,
     return x;
 }
 
+/* Helper for Computing the linear offset into an 4-dimensional ggml_tensor given
+per-dimension sizes, strides, and indices */
+template<int N>
+static __dpct_inline__ size_t calculate_offset(const std::array<int, N> & strides, const std::array<int, N> & indices) {
```

Comment:
Nit but I don't think it makes sense to mark the function as `static` in a header, it's just going to be ignored.

Reply:
There are other functions like `warp_reduce_sum` and `warp_reduce_max` which use static, so I was confused.